### ISSUE TITLE
Fix back the sorting message

### ISSUE THREAD

**Issue by curquiza** [meilisearch/meilisearch#3722]
Seems like we had a regression.
Should be fixed for v1.2

More info: https://github.com/meilisearch/meilisearch/discussions/1713#discussioncomment-5793584

@Kerollmops

**Comment by ManyTheFish** [comment]
Hello @curquiza this fix will be merged with a different error message, this is a tiny change, but I don't know if we have to notify the docs team about it, I put below the change:

#### Reintroduce fix (previous and expected behavior)
```
You must specify where "sort" is listed in the rankingRules setting to use the sort parameter at search time
```
#### Replace double-quotes with back-ticks (new behavior validated with the product team)
```
You must specify where `sort` is listed in the rankingRules setting to use the sort parameter at search time
```

**Comment by curquiza** [comment]
Thank you many for the information. No, the docs team will not be impacted by this change

I close this issue, fixed by #3749, will be released in rc1 during this week
